great_expectations/init_notebooks/sql/create_expectations.ipynb

```diff
@@ -0,0 +1,357 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "context = ge.data_context.DataContext()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. List the tables in your database.\n",
+    "\n",
+    "The `DataContext` will now introspect your database (a `Datasource`) and list the tables. [Read more in the tutorial](https://docs.greatexpectations.io/en/latest/getting_started/create_expectations.html?utm_source=notebook&utm_medium=create_expectations#data-assets)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "great_expectations.jupyter_ux.list_available_data_asset_names(context)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. Pick a table and set the expectation suite name\n",
+    "\n",
+    "Internally, Great Expectations represents tables as `DataAsset`s and uses this notion to link them to `Expectation Suites`. To learn more about `DataAssets` and how their names are built, see [the reference](https://docs.great_expectations.io/en/latest/reference/data_context_reference.html#data-asset-names). "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "table_name = \"YOUR_TABLE\"  # TODO: replace with your value!\n",
+    "data_asset_name = context.normalize_data_asset_name(table_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We recommend naming your first expectation suite for a table `warning`. Later, as you identify some of the expectations that you add to this suite as critical, you can move these expectations into another suite and call it `failure`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "expectation_suite_name = \"warning\" # TODO: replace with your value!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Create a new empty expectation suite"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "context.create_expectation_suite(data_asset_name=data_asset_name, expectation_suite_name=expectation_suite_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 5. Load a batch of data you want to use to create `Expectations`\n",
+    "\n",
+    "To learn more about `get_batch` with other data types (such as csv files, pandas, or Spark), see [this tutorial](https://docs.greatexpectations.io/en/latest/getting_started/create_expectations.html?utm_source=notebook&utm_medium=create_expectations#get-batch)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# If you would like to load an entire table or view:\n",
```

Comment:
```suggest
"# If you would like to load an entire table or view in the default schema:\n",
```

Reply:
I want to make this point stronger, because even if the asset name is "someschema.sometable" and you are specifying table_name to be "someschema.sometable", this is not going to work in get_batch. In this case even though you are using the same table that the generator sees, you have to specify table and schema separately.

Reply:
Fixed.